src/doc/reference.md

```diff
@@ -2959,10 +2956,10 @@ meaning of the operators on standard types is given here.
   : Exclusive or.
     Calls the `bitxor` method of the `std::ops::BitXor` trait.
 * `<<`
-  : Logical left shift.
+  : Aritmetic left shift.
```

Comment:
>  The "Arithmetic" versus "Logical" distinction does not arise for the left shift operator

I _thought_ this was true, but since they both said it, I just assumed I was the one in error. Thanks for the summary, fixing now.